Source/Shaders/Builtin/Functions/acesTonemapping.glsl

```diff
@@ -0,0 +1,15 @@
+vec3 czm_acestoneMapping(vec3 color)
```

Comment:
Should be `czm_acesTonemapping` to match the file name

Reply:
This was pushed too early